.github/workflows/integ_test_datastore_v2.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
   datastore-integration-v2-test:
     continue-on-error: true
-    timeout-minutes: 20
+    timeout-minutes: 17
```

Comment:
I can see this successful test run here https://github.com/aws-amplify/amplify-swift/actions/runs/3413375249/jobs/5680024350 that ran for 14 mins, and a very short "prepare-for-test" step that took 20 seconds.. I would say 20 or 25 seems like a reasonable amount of time to build a buffer for when it runs long. And less than 30 minutes so when something hangs, it times out and fails the test run so it's not hanging forever. regarding the test run, 14 minutes still seems awfully high though.. keeping the timeout higher will help us capture more data points on a couple of runs. If the test doesn't have a syncExpression to filter out data created before the current test and it's waiting for sync, then that might be contributing to the overall time. If this is what's happening (i haven't looked into these tests to say definitively) but let's say it takes 30 seconds to fully sync the backend data and there are 20 tests, each test clears and resyncs is going to be at least 10 mins